documentation/meta/incidents/runbooks/incident_response_runbook.md

```diff
@@ -0,0 +1,120 @@
+# Incident Response Runbook
+
+```{note}
+**Recording** an incident is a standalone task which can be performed by any maintainer. This recorder may not stabilize the service or continue to work on incident triage.
+```
```

Comment:
Should we add a clarification that this runbook is mostly for mantainers use? Given it's referring to internal tools and concepts as MSR and P2, it's probably easier leave it as is than to try to rephrase the terms with more generic and difuse alternatives, like "internal blog" instead of P2 and so on.